src/auto/subprocess.rs

```diff
@@ -0,0 +1,492 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use Cancellable;
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use Error;
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use InputStream;
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use OutputStream;
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use SubprocessFlags;
+use ffi;
+#[cfg(feature = "futures")]
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use futures_core;
+use glib;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use std;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct Subprocess(Object<ffi::GSubprocess>);
+
+    match fn {
+        get_type => || ffi::g_subprocess_get_type(),
+    }
+}
+
+impl Subprocess {
+    //#[cfg(any(feature = "v2_40", feature = "dox"))]
+    //pub fn new<'a, P: Into<Option<&'a Error>>>(flags: SubprocessFlags, error: P, argv0: &str, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs) -> Subprocess {
```

Comment:
I have a crate that we could use to give `VarArgs`: https://crates.io/crates/va_list-rs (apparently, you can just give a `va_list` as a `VarArgs` argument so why not enjoying this?). Should we give it a try?

Reply:
IMHO it worth to try but not in this PR (or if PR not merged until next release)

Reply:
Also we discussed that a while ago on IRC and I believe there are some safety concerns there. Have to look again, I can't remember :)

Reply:
I don't intend to put it in this PR but since I had the chance to talk about it... 😛

Reply:
Also the `newv` variant is equivalent and more Rust-y :)